sprint0/sprint0/HUD/HUDItem.cs

```diff
@@ -23,22 +23,20 @@ public HUDItem(Texture2D texture, Vector2 location)
         }
         public void Draw(SpriteBatch spriteBatch)
         {
-            if (Item != PlayerItems.None && SmallItem()) spriteBatch.Draw(Texture, new Rectangle(Location.X, Location.Y, (int)(Width * Game1.Scale), (int)(Height * Game1.Scale)), ItemMap[Item], Color.White);
-            else if (Item != PlayerItems.None) spriteBatch.Draw(Texture, new Rectangle((Location.X - Width - 1), Location.Y, (int)(Width * 2 * Game1.Scale), (int)(Height * Game1.Scale)), ItemMap[Item], Color.White);
+            if (Item != PlayerItems.None && SmallItem())
+                spriteBatch.Draw(Texture, new Rectangle(Location.X, Location.Y, (int)(Width * Game1.Scale), (int)(Height * Game1.Scale)), ItemMap[Item], Color.White);
+            else if (Item != PlayerItems.None)
+                spriteBatch.Draw(Texture, new Rectangle((Location.X - Width - 1), Location.Y, (int)(Width * 2 * Game1.Scale), (int)(Height * Game1.Scale)), ItemMap[Item], Color.White);
         }
 
-        public void Update()
-        {
-        }
+        public void Update() { }
 
         public void SetItem(PlayerItems item)
         {
-            if (ItemMap.ContainsKey(item)) Item = item;
+            if ((ItemMap.ContainsKey(item) && Item == PlayerItems.None) || item == PlayerItems.None)
```

Comment:
did you mean `Item != PlayerItems.None` for the first part

Reply:
or I think we don't even need it since if it's none, itll evaluate to true anyways.